meta-lmp-base/files/lmp-group-table

```diff
@@ -69,5 +70,7 @@ systemd-bus-proxy:x:991:
 systemd-timesync:x:992:
 systemd-resolve:x:993:
 systemd-journal:x:994:
+pulse-access:x:995:
+pulse-rt:x:996:
 fio:x:1000:
-nogroup:x:65534:
+nogroup:x:65534:
```

Comment:
Add newline back to the end of the line.

Reply:
done!